### ISSUE TITLE
measuring the distance

### ISSUE THREAD

**Issue by urbansound8K** [maudzung/SFA3D#34]
do you have any idea how to measuring the distance of detected objects? I mean instate of presenting the FPS I want to display the distance of an object?

**Comment by maudzung** [comment]
Hi @urbansound8K 
For each detected object, we have information of its box:
- Center position  `x`, `y`, `z`, 
- Size: `sx`, `sy`, `sz`
- Yaw angle

You can compute `distance = np.sqrt(x**2 + y**2 + z**2)`, then use `OpenCV` to write the information to the corresponding images

**Comment by urbansound8K** [comment]
Hey there
Thank you very much for replying and your great job. 

when I measured and disapplied the distance I got wired results.  it gives me extra results every time I display a new scene. to break it down, image that I have a scene with two objects, but the output of it is 5 values. Also, it display the last results only. 

![lidarpng](https://user-images.githubusercontent.com/56126016/112096921-799ead80-8b32-11eb-8de7-00cd26847524.png)
